### PR TITLE
[Managed] Test updates for deleting ApplicationKey, LogoutAsync

### PR DESCRIPTION
**Pull request by pragnagopa**

### CODE REVIEW

**Comment by phvannor** [comment]
You seem to also have diffs showing daylight changes, can you rebase onto MobileApp?  Or are you bringing them yourself over from master?

File: sdk/managed/test/Microsoft.WindowsAzure.MobileServices.WindowsStore.Test/Functional/OfflineTests.cs [hunk 1]
```diff
@@ -214,7 +214,7 @@ public class OfflineTests : FunctionalTestBase
             Log("Using random seed: {0}", seed);
             Random rndGen = new Random(seed);
 
-            var offlineReadyClient = CreateClient();
+            var offlineReadyClient = await CreateClient();
```
Comment (phvannor) [inline sdk/managed/test/Microsoft.WindowsAzure.MobileServices.WindowsStore.Test/Functional/OfflineTests.cs]: can we rename createClient to createClientAsync() then?  Why is it async?

File: sdk/managed/test/Microsoft.WindowsAzure.MobileServices.WindowsStore.Test/Functional/OfflineTests.cs [hunk 2]
```diff
@@ -400,7 +398,7 @@ private MobileServiceClient CreateClient(params HttpMessageHandler[] handlers)
             }
             else
             {
-                offlineReadyClient.Logout();
+                await offlineReadyClient.LogoutAsync();
```
Comment (phvannor) [inline sdk/managed/test/Microsoft.WindowsAzure.MobileServices.WindowsStore.Test/Functional/OfflineTests.cs]: why do we need to call logout on client init?

File: sdk/managed/test/Microsoft.WindowsAzure.MobileServices.Android.Test/LoginActivity.cs [hunk 3]
```diff
@@ -16,16 +16,10 @@ public class LoginActivity : Activity
...
             public const string MobileServiceUri = "MobileServiceUri";
-            public const string MobileServiceKey = "MobileServiceKey";
             public const string TagExpression = "TagExpression";
 
             public const string AutoStart = "AutoStart";
-            public const string MasterRunId = "MasterRunId";
             public const string RuntimeVersion = "RuntimeVersion";
-            public const string CliendId = "CliendId";
-            public const string ClientSecret = "ClientSecret";
-            public const string DayLightUrl = "DayLightUrl";
```
Comment (pragnagopa) [inline sdk/managed/test/Microsoft.WindowsAzure.MobileServices.Android.Test/LoginActivity.cs]: Xamarin solution is missing changes for daylight. Will submit another PR to master branch

File: sdk/managed/test/Microsoft.WindowsAzure.MobileServices.TestFramework/Framework/MasterTestResult.cs [hunk 4]
```diff
@@ -1,39 +1,39 @@
...
-
-        [JsonProperty("reference_url")]
+        public DateTime EndTime { get; set; }
+
+        [JsonProperty("reference_url")]
         public string ReferenceUrl { get; set; }
-    }
-}
+    }
+}
```
Comment (dhei) [inline sdk/managed/test/Microsoft.WindowsAzure.MobileServices.TestFramework/Framework/MasterTestResult.cs]: You don't need to change this file. It has no real changes except whitespaces.
